shared_model/schema/commands.proto

```diff
@@ -125,6 +125,18 @@ message CallEngine {
     string input = 4;  // hex string
 }
 
+message DataModelId {
+    string name = 1;
+    string version = 2;
+}
```

Comment:
I think this should better be in `primitive.proto`, in case we will reuse it in queries